### ISSUE TITLE
How to calculate population prevalence and sample prevalence?

### ISSUE THREAD

**Issue by Steven-ZWQ** [cuelee/pleio#21]
As Table S1 in the original article, both sample prevalence and population prevalence equaled the number of cases divided by the number of controls when data were obtained from a population-based cohort (UK Biobank). The definition of "prevalence" in pleio seems to be different from the terminology in the epidemiology, which may cause ambiguity. I'm not sure whether my understanding is correct. Thanks!

**Comment by cuelee** [comment]
Yes, it is correct. 

We estimated the disease prevalence obtained from a population-based cohort (UK Biobank) as: N case / (N case + N control)

**Comment by cuelee** [comment]
Yes, you are correct. The prevalence in Table S1 uses case/control ratio. In your analysis, please uses case/population instead case-control ratio.

**Comment by cuelee** [comment]
With regard to this question, please refer to https://doi.org/10.1002/gepi.21614.

**Comment by Steven-ZWQ** [comment]
Thanks.
